<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="unittest.loader._FailedTest-20220210133119" tests="1" file="unittest/loader.py" time="0.002" timestamp="2022-02-10T13:31:19" failures="0" errors="1" skipped="0">
	<testcase classname="unittest.loader._FailedTest" name="circle_tests" time="0.002" timestamp="2022-02-10T13:31:19">
		<error type="ImportError" message="Failed to import test module: circle_tests
Traceback (most recent call last):
  File &quot;c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\loader.py&quot;, line 154, in loadTestsFromName
    module = __import__(module_name)
  File &quot;c:\users\thefastfamily\nextcloud\beuth-myhome\m02 advanced softwareengineering\04 projects\sandbox_pycharm\src\unittest\python\circle_tests.py&quot;, line 4, in &lt;module&gt;
    from src.main.python.physics_engine.circle import Circle
ModuleNotFoundError: No module named 'src'
"><![CDATA[Traceback (most recent call last):
  File "c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\case.py", line 60, in testPartExecutor
    yield
  File "c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\case.py", line 633, in _callTestMethod
    method()
  File "c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\loader.py", line 34, in testFailure
    raise self._exception
ImportError: Failed to import test module: circle_tests
Traceback (most recent call last):
  File "c:\users\thefastfamily\appdata\local\programs\python\python38\lib\unittest\loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "c:\users\thefastfamily\nextcloud\beuth-myhome\m02 advanced softwareengineering\04 projects\sandbox_pycharm\src\unittest\python\circle_tests.py", line 4, in <module>
    from src.main.python.physics_engine.circle import Circle
ModuleNotFoundError: No module named 'src'

]]></error>
	</testcase>
</testsuite>
